store/postgres/src/relational.rs

```diff
@@ -376,6 +376,14 @@ impl Layout {
             Some(skip.to_string())
         };
 
+        let order = match (order, tables.iter().next()) {
```

Comment:
Would be good to comment on why you can look only at the first table, also this match suggests `tables` might be empty, is that possible? If not we should error or panic in that case.

Reply:
Added an explanation. Commit 5dc5aa08